### ISSUE TITLE
CMS is not accessible

### ISSUE THREAD

**Issue by AccessAbilityOfficer** [directus/directus#22512]
### Describe the Bug

The issue I personally run into is the lack of visual keyboard support...
can you tell which button has focus?
![CMS lack visual focus](https://github.com/directus/directus/assets/145735935/2c7c625c-b408-427d-8c31-20b2260b4e9f)

It's the back button... it's a darker grey than when it doesn't have focus... but using shades of grey is a little crazy because I though it was disabled.  The Save button is no better and there are several elements with NO visual change (the FLX logo and the active tab).    Could you please do a quick and dirty CSS change to force a focus rectangle around the keyboard focus of all elements?

It's also a persona pain that to save this form I must navigate back up through it to get to the Save button, or give up and switch to using the mouse 🙁

But to make sure I was providing accurate information I turned on NVDA (a free and popular screen reader) to confirm where the keyboard focus was I found controls with No programmatic names.  Has this ever had an accessibility evaluation?  doing a quick internet search i couldn't find a VPAT - could someone from Sales contact me (ADA@AccessAbilityOfficer.com)

### To Reproduce

open - https://cms.flxwebsites.com/admin/content/accessability_officer_articles/7
using the tab key to navigate the page

### Directus Version

10.10.4

### Hosting Strategy

Self-Hosted (Custom)

**Comment by br41nslug** [comment]
> It's also a persona pain that to save this form I must navigate back up through it to get to the Save button, or give up and switch to using the mouse 🙁

To address this specific point you could also the keyboard shortcuts to save.
![image](https://github.com/directus/directus/assets/9389634/ed4df23c-f470-4071-8f1a-e3bb1a0acdb9)

**Comment by rijkvanzanten** [comment]
@AccessAbilityOfficer that doesn't mean it won't happen or isn't important. In our bigger roadmap it makes more sense for us to combine this with other to-be-announced efforts on the app side rather than trying to retroactively add this in 👍🏻

**Comment by A11yHeather** [comment]
@rijkvanzanten - Just to be clear because our Website hosting/design company uses Directus we at AccessAbility Officer have to shift work around so that our sighted mouse users are the only ones contributing to our website :(  our low vision and blind team members (which is most of our company) are unable to use the UI to post jobs or add Blog post.   

I realize that software development takes time and that companies don't like to announce efforts before they are almost baked but in this case can you please help us understand when we might be able to look at a persons role rather than their proficiency with a mouse in making assignments?

**Comment by rijkvanzanten** [comment]
It's difficult to give any real timelines as things are actively being planned out and developed, but we're working on an overhauled component library that focusses on accessibility from the ground up which should help a ton here. I'd expect some low hanging fruit to be resolved in the coming weeks/months with a more impactful change in about 6 months

**Comment by micheleferraro** [comment]
Hi, I'm following this.
I think it's a big error to consider accessibility a feature an put it in a priority queue.
Accessibility should always be part of the design process.
Since accessibility will be mandatory for digital products in Italy in 2025 we'll be forced to move to other CMS unfortunately...

**Comment by rijkvanzanten** [comment]
@micheleferraro Yup! And it has been for new work. That being said, we do have to plan for upgrading the older pieces to match the new workflow, which is a one off (large) piece of work 👍🏻

**Comment by micheleferraro** [comment]
Thank you @rijkvanzanten, I know it's not easy and I'm really happy to hear you're working on it!

**Comment by micheleferraro** [comment]
Hi, any news on this topic?
The European Accessibility Act states that by 2025 accessibility will be required for new products in all European countries.
In Italy, the deadline is June 28th, after this date, it will essentially no longer be possible to use Directus…
